UOP1_Project/Assets/Scripts/Characters/StateMachine/Actions/FaceProtagonistSO.cs

```diff
@@ -0,0 +1,37 @@
+﻿using UnityEngine;
+using UOP1.StateMachine;
+using UOP1.StateMachine.ScriptableObjects;
+
+[CreateAssetMenu(fileName = "FaceProtagonist", menuName = "State Machines/Actions/Face Protagonist")]
+public class FaceProtagonistSO : StateActionSO
+{
+	protected override StateAction CreateAction() => new FaceProtagonist();
+}
+
+public class FaceProtagonist : StateAction
+{
+	Transform _protagonist;
+	Transform _actor;
+
+	public override void Awake(StateMachine stateMachine)
+	{
+		_actor = stateMachine.transform;
+	}
+		
+	public override void OnUpdate()
+	{
+		if (_protagonist != null)
+		{
+			_actor.LookAt(_protagonist);
```

Comment:
Note that this is creating some probably undesired behavior!
![image](https://user-images.githubusercontent.com/27371812/100672241-67f58800-3340-11eb-9c96-6a474485e765.png)
😆 
Rotation should be only on the Y axis.